.github/workflows/linux_builds.yml

```diff
@@ -72,6 +72,16 @@ jobs:
             # Skip 2GiB artifact speeding up action.
             artifact: false
 
+          - name: Editor w/ C++20 (target=editor, tests=yes, dev_build=yes, cpp_standard=20)
```

Comment:
Do we really want to add two builds though? It would take up much resources just for an edgecase that could be tested (not even needs to be tested regularly) by someone.

Reply:
Hmm, that's fair. I could just assign that value to an existing job instead, because it being accounted for in *some* form is important